src/main.rs

```diff
@@ -189,18 +189,39 @@ fn actual_main(paths: ForemanPaths) -> ForemanResult<()> {
             .init();
     }
 
-    match options.subcommand {
+    let subcommand_result: ForemanResult<()> = match options.subcommand {
```

Comment:
Since you are using `return Err(...)` within the match, you'll always get an `Ok(())` in that variable. So you can revert it to how it was before:

```rust
match options.subcommand {
    // ...
}

Ok(())
```